notebooks/datasets.ipynb

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
Do we really want to be making our notebooks public in this repo?  I suggest we take these out and create a separate (possibly private) repo with our notebooks.

Reply:
Testing.ipynb should be removed (just did that), but demo.ipynb and datasets.ipynb are meant to be tutorial-ish notebooks that make it easier for people to understand components in the code, so they should be here

Reply:
@Hollyqui the only concern is that .ipynb files contain a lot of wrappers and metadata lines, which don’t work well with git. For example, re-running cells will mark the .ipynb file as a changed file from git’s perspective due to the modified notebook metadata. A better approach would be to provide examples as simple python scripts unless there’s a need to show charts